benchmarks/benchmark_serving.py

```diff
@@ -171,9 +171,9 @@ async def benchmark(
     else:
         raise ValueError(f"Unknown backend: {backend}")
 
-    pbar = None if disable_tqdm else tqdm(total=len(input_requests))
-
     print(f"Traffic request rate: {request_rate}")
+    
```

Comment:
Please remove the whitespace in this line to fix the `yapf` format error. Thanks!

Reply:
The whitespace is removed now